<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KalignDialog</class>
 <widget class="QDialog" name="KalignDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>505</width>
    <height>222</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Align with Kalign</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Advanced options</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="12" column="0">
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="1" column="1">
         <widget class="QDoubleSpinBox" name="gapOpenSpinBox">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>0</height>
           </size>
          </property>
          <property name="value">
           <double>54.950000000000003</double>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>255</width>
            <height>17</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="0">
         <widget class="QCheckBox" name="gapOpenCheckBox">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>Gap open penalty</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QCheckBox" name="gapExtensionPenaltyCheckBox">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>Gap extension penalty</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QDoubleSpinBox" name="gapExtensionPenaltySpinBox">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>0</height>
           </size>
          </property>
          <property name="value">
           <double>8.520000000000000</double>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>255</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="3" column="0">
         <widget class="QCheckBox" name="terminalGapCheckBox">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>Terminal gap penalty</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QDoubleSpinBox" name="terminalGapSpinBox">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>0</height>
           </size>
          </property>
          <property name="value">
           <double>4.420000000000000</double>
          </property>
         </widget>
        </item>
        <item row="3" column="2">
         <spacer name="horizontalSpacer_5">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>255</width>
            <height>17</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="4" column="0">
         <widget class="QCheckBox" name="bonusScoreCheckBox">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>Bonus score</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QDoubleSpinBox" name="bonusScoreSpinBox">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>0</height>
           </size>
          </property>
          <property name="value">
           <double>0.020000000000000</double>
          </property>
         </widget>
        </item>
        <item row="4" column="2">
         <spacer name="horizontalSpacer_6">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>255</width>
            <height>17</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="13" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:8pt; color:#3d3d3d;&quot;&gt;*Note that default values for unchecked options may differs dependent from alphabet of alignment.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>3</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="alignButton">
       <property name="text">
        <string>Align</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>alignButton</sender>
   <signal>clicked()</signal>
   <receiver>KalignDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>411</x>
     <y>178</y>
    </hint>
    <hint type="destinationlabel">
     <x>252</x>
     <y>139</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>KalignDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>494</x>
     <y>178</y>
    </hint>
    <hint type="destinationlabel">
     <x>252</x>
     <y>139</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bonusScoreCheckBox</sender>
   <signal>clicked(bool)</signal>
   <receiver>bonusScoreSpinBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>54</x>
     <y>119</y>
    </hint>
    <hint type="destinationlabel">
     <x>196</x>
     <y>125</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>terminalGapCheckBox</sender>
   <signal>clicked(bool)</signal>
   <receiver>terminalGapSpinBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>62</x>
     <y>92</y>
    </hint>
    <hint type="destinationlabel">
     <x>164</x>
     <y>99</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>gapExtensionPenaltyCheckBox</sender>
   <signal>clicked(bool)</signal>
   <receiver>gapExtensionPenaltySpinBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>60</x>
     <y>67</y>
    </hint>
    <hint type="destinationlabel">
     <x>174</x>
     <y>73</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>gapOpenCheckBox</sender>
   <signal>clicked(bool)</signal>
   <receiver>gapOpenSpinBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>38</x>
     <y>40</y>
    </hint>
    <hint type="destinationlabel">
     <x>174</x>
     <y>46</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
